### PR TITLE
feat(umbraco_mu): new service for umbraco in MU

### PR DESCRIPTION
**Pull request by stavamichal**
* exporting information about resources and users' logins

### CODE REVIEW

File: gen/umbraco_mu [hunk 1]
```diff
@@ -0,0 +1,53 @@
+#!/usr/bin/perl
+
+use strict;
+use warnings;
+use perunServicesInit;
+use perunServicesUtils;
+use File::Basename;
+use perunServicesInit;
```
Comment (zlamalp) [inline gen/umbraco_mu:8]: Duplicate import

File: gen/umbraco_mu [hunk 2]
```diff
@@ -0,0 +1,53 @@
...
+
+my $siteStruc = {};
+
+foreach my $resourceId ($data->getResourceIds()) {
+	my $groupName = $data->getResourceAttributeValue(resource => $resourceId, attrName => $A_RESOURCE_UMBRACO_GROUP_NAME );
+	next unless $groupName;
+
+	foreach my $memberId ($data->getMemberIdsForResource( resource => $resourceId )) {
+		my $login = $data->getUserFacilityAttributeValue( member => $memberId, attrName => $A_USER_LOGIN );
+		$siteStruc->{$groupName}->{$login} = $login;
```
Comment (zlamalp) [inline gen/umbraco_mu:30]: You can probably just store `1` as a value and not login, since you use only hash keys later.

File: gen/umbraco_mu [hunk 3]
```diff
@@ -0,0 +1,53 @@
...
+	my $record = {};
+	$record->{'groupName'} = $groupName;
+	my @members = ();
+	foreach my $login (sort keys %{$siteStruc->{$groupName}}) {
+		push @members, $login;
+	}
+	$record->{'members'} = \@members;
+	push @printingStruc, $record;
+}
+
```
Comment (zlamalp) [inline gen/umbraco_mu:46]: You should probably sort also groups by their names in order to get comparable output:
```perl
@printingStruc = sort { $a->{'groupName'} cmp $b->{'groupName'} } @printingStruc;
```

**Review by zlamalp — CHANGES_REQUESTED** [review]

File: gen/umbraco_mu (hunk 1 above)
Comment (stavamichal) [reply gen/umbraco_mu]: Removed.

File: gen/umbraco_mu (hunk 2 above)
Comment (stavamichal) [reply gen/umbraco_mu]: Changed.

File: gen/umbraco_mu (hunk 3 above)
Comment (stavamichal) [reply gen/umbraco_mu:47]: Added.

**Review by zlamalp — APPROVED** [review]

**Pull request merged**